SQL/migrate-2023/V004__suit_sensors.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Implemented in PR #17607.
+-- Adds suit sensor prefs.
+--
+ALTER TABLE `ss13_characters` ADD `sensor_setting` INT(11) DEFAULT NULL AFTER `headset_choice`;
+ALTER TABLE `ss13_characters` ADD `sensors_locked` INT(11) DEFAULT NULL AFTER `sensor_setting`;
```

Comment:
Suit Sensors should not be lockable via the char setup menu.
Not every antag might want to completely strip players to prevent them from being tracked, which what would require.

Reply:
sensor lock has been removed